translate/storage/pypo.py

```diff
@@ -804,11 +810,17 @@ def addcomment(thepo):
                         markedpos.append(thepo)
                     thepo.msgctxt.append('"%s"' % escapeforpo(" ".join(thepo.getlocations())))
                     uniqueunits.append(thepo)
+                else:
```

Comment:
the reason i didnt was that anything you put here would discard it anyway just not trigger the warning